src/ell/models/anthropic.py

```diff
@@ -28,6 +28,8 @@ def register(client: anthropic.Anthropic):
             ('claude-3-sonnet-20240229', 'anthropic'),
             ('claude-3-haiku-20240307', 'anthropic'),
             ('claude-3-5-sonnet-20240620', 'anthropic'),
+            ('claude-3-5-sonnet-20241022', 'anthropic'),
+            ('claude-3-5-sonnet-latest', 'anthropic'),
```

Comment:
i dont think latest is an identifier in the api can you confirm this or delete it?

Reply:
It is the preferred way to use latest see: https://docs.anthropic.com/en/docs/about-claude/models

<img width="386" alt="image" src="https://github.com/user-attachments/assets/c3e03c6a-5561-4fbd-b6ec-a5dcc5a8779d">